### PR TITLE
Create branch 3.x for updates to modern PHP

### PR DESCRIPTION
**Pull request by cxj**
Branch 3.x is for PHP 7.3 and 7.4, as well as PHPUnit 7, 8 and 9 updates.

### CODE REVIEW

**Comment by cxj** [comment]
Ah yes, the Travis configuration is very out of date.  That should be updated to make it compatible with modern PHP.  Will append the needed changes to this PR.

**Comment by harikt** [comment]
Hi @cxj,

I have created a 3.x branch. Also added you to collaborator into this repo. I didn't find a way to add you to team. May be I have less permissions.

Keep your good work into this repo.

Thank you.

**Comment by harikt** [comment]
Don't merge this to 2.x. Push to 3.x . I am not closing for you can take the necessary steps to move this forward.

File: config/Common.php [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?php
 namespace Aura\Cli\_Config;
 
-use Aura\Di\Config;
+use Aura\Di\ContainerConfig as Config;
```
Comment (harikt) [inline config/Common.php:4]: 3.x probably don't need these container config I believe.

**Comment by cxj** [comment]
Hi Hari!  Thanks for adding me to the collaborators and creating this branch.  I will follow your suggestions.

**Comment by kenjis** [comment]
@pmjones @harikt @koriym
This PR is for PHP 7.x. So it should be merged into 4.x.
See https://github.com/auraphp/Aura.Cli_Kernel/pull/12#issuecomment-873248524
Can you create `4.x` branch?

**Comment by koriym** [comment]
@pmjones I would like to confirm that

The Aura 3.x series was for PHP 5.6 and later.
The Aura 4.x series is for PHP 7.2 and later.

Is that correct? There is nothing in 3.x right now.
Do we still need 4.x?

https://github.com/auraphp/Aura.Cli_Kernel/pull/12#issuecomment-873248524

**Comment by cxj** [comment]
Branch 3.x _in this repo_ is for PHP 7.3 and 7.4, as well as PHPUnit 7, 8 and 9 updates.

**Comment by harikt** [comment]
Hey @cxj , the one you merged seems to be going to 2.x . Is this by accident ?

**Comment by harikt** [comment]
@kenjis I have created 4.x branch for you. Please note the 4.x is created from 3.x . @koriym regarding your question about why 4.x : Earlier some where iirc @pmjones wished not to confuse people. So when they hear 4.x of aura they can easily guess this is 7.x , so 3.x -> 5.x so and so.

**Comment by cxj** [comment]
@harikt  It was meant for 3.x.  If I merged to 2.x, it was an accident and a mistake, yes.  Sorry.

**Comment by harikt** [comment]
I have reverted this merge.

**Comment by kenjis** [comment]
@harikt 
> So when they hear 4.x of aura they can easily guess this is 8.x , so 3.x -> 7.x so and so.

In my understanding:
- The Aura 3.x series was for PHP 5.6 and later.
- The Aura 4.x series is for PHP 7.2 and later.

to **all** Aura repositoies.

So this PR should go to `4.x`, not `3.x`.
Am I misunderstanding?

**Comment by harikt** [comment]
@kenjis you are right. 

> The Aura 4.x series is for PHP 7.2 and later.

**Comment by kenjis** [comment]
@cxj Could you send this PR to `4.x` once more?

- `4.x` is for PHP 7.2 or later
- PHPUnit `^8.5` is better, because it supports PHP8 and Xdebug3. See https://github.com/auraphp/Aura.Project_Kernel/pull/19

**Comment by kenjis** [comment]
@cxj I will send a new PR that includes this PR to `4.x`.

**Pull request merged**